site/src/docs/components/navigation-link/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+slug: '/components/navigation-link'
+title: 'NavigationLink'
+navTitle: 'NavigationLink'
+---
+
+import { NavigationLink } from 'hds-react';
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<PlaygroundPreview>
+  <NavigationLink href="#">Link</NavigationLink>
+</PlaygroundPreview>
+
+### Principles
+
+- **Navigation link component is meant to be used inside the header and footer components!** If you're looking for inline links, see the documentation about <InternalLink href="/components/link">HDS link components</InternalLink>.
+- Use navigation link component only inside <InternalLink href="/components/header-universal-bar">HeaderUniversalBar</InternalLink>, Navigation and Footer components.
+- Because navigation links are designed for the purpose of site navitation, they should not open in a new tab. However this may be desired if there is an external link in the universal bar or in the footer.
```

Comment:
Grammarly suggestion: "Because navigation links are designed for site navigation, they should not open in a new tab. However, this may be desired if there is an external link in the universal bar or the footer."